src/windows/mod.rs

```diff
@@ -578,3 +589,16 @@ cfg_if! {
         // Unknown target_env
     }
 }
+
+pub fn kill(pid: pid_t, sig: c_int) -> ::c_int {
+    let dwDesiredAccess = PROCESS_TERMINATE;
+    let bInheritHandle: winapi::shared::minwindef::BOOL = false.into();
+    let dwProcessId = pid as _;
+    let result;
+    unsafe {
+        let hProcess = OpenProcess(dwDesiredAccess, bInheritHandle, dwProcessId);
```

Comment:
**unlikely** to be merged but wherever you implement it, I would recommend to check hProcess value before calling TerminateProcess/CloseHandle with.

Reply:
Fixed missing check for hProcess value